admin_guide/sdn_troubleshooting.adoc

```diff
@@ -630,6 +630,23 @@ provide the output from the debugging script)
 * Kube - declare as a LoadBalancer and something _you_ have to write does the rest
 * OS/AE - Both use the router
 
+[[tls-handshake-timeout]]
+=== TLS Handshake Timeout
+
+When a pod fails to deploy, check its docker log for a TLS handshake timeout:
+
+----
+$ docker log <container_id>
+...
+[...] couldn't get deployment [...] TLS handshake timeout
```

Comment:
It's supposed represent elided text, unimportant for the task of recognizing this error condition.  How about i drop the square-braces and just use `...` for that?